src/ui/Templates/CoreHelpers.ts

```diff
@@ -297,15 +312,36 @@ export interface ICoreHelpers {
 
 /**
  * Available options for the size templateHelpers.
+ *
+ * Example:
```

Comment:
I would remove this example. As discussed, it makes more sense to have usage examples in the helper documentation than in its options interface.